package.json

```diff
@@ -72,6 +72,9 @@
     "prettier": "2.4.1",
     "rewire": "5.0.0"
   },
+  "resolutions": {
+    "@typescript-eslint/typescript-estree": "5.3.0"
+  },
```

Comment:
taken from here: https://github.com/facebook/create-react-app/issues/7255

Reply:
This until CRA will update it's own..

Reply:
yep